samples/winapi_drivelist/main.c

```diff
@@ -7,27 +7,30 @@ int main(void)
 {
     XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
+    // Create a variable to check if there was any error in the program
+    BOOL errored = false;
     // Mount some drives for demonstration purposes
     BOOL ret;
     ret = nxMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
     if (!ret) {
-        debugPrint("Failed to mount C: drive!\n");
-        Sleep(5000);
-        return 1;
+        errored = true;
```

Comment:
This should probably be done after handling the error, to not disturb the readability of the important bits.

Also if this is later turned into a `mark_error();` (for whatever reason) then that function might actually change the `GetLastError()` value.